app/src/main/java/com/edu/sun/oereminder/data/source/remote/model/Member.kt

```diff
@@ -0,0 +1,8 @@
+package com.edu.sun.oereminder.data.source.remote.model
+
+data class Member(
+    val accountId: Int,
+    val avatarImageUrl: String,
```

Comment:
```suggestion
    val avatarUrl: String,
```
avatar nó đã là ảnh r nên k cần Image đâu ô =))